src/spacel/provision/template/app_spot.py

```diff
@@ -21,13 +22,23 @@ def spotify(self, app, region, template):
             return
 
         resources = template['Resources']
-        self._add_spot_fleet(app, region, resources)
+        parameters = template['Parameters']
+        self._add_spot_fleet(app, region, resources, parameters)
         self._clean_up_asg(template)
 
     @staticmethod
-    def _add_spot_fleet(app, region, resources):
+    def _add_spot_fleet(app, region, resources, parameters):
         spot_price = app.spot.get('price', '1.00')
 
+        # Set Name tag
+        tags = {
+            'Name': {'Ref': 'AWS::StackName'}
+        }
+        user_data_param = parameters['UserData']['Default'] or ''
+        if user_data_param:
+            user_data_param += ','
+        user_data_param += '"tags":' + json.dumps(tags)
```

Comment:
Fair point, I did not consider that! I see that the StackName (https://github.com/pebble/spacel-provision/blob/master/src/spacel/provision/cloudformation.py#L103 originating from https://github.com/pebble/spacel-provision/blob/master/src/spacel/provision/provision.py#L22 `stack_name`) actually equals the `app` variable here. Instead of injecting, should I be safe to just use `app` instead?